src/mudlet.cpp

```diff
@@ -1217,9 +1241,13 @@ bool mudlet::openWindow(Host* pHost, const QString& name, bool loadLayout)
         pC->layerCommandLine->hide();
         pC->mpScrollBar->hide();
         pC->setUserWindow();
+        pC->console->setIsMiniConsole();
+        pC->console2->setIsMiniConsole();
```

Comment:
Note that the `TTextEdit::mIsMiniConsole` flag is different from the `TConsole::mIsSubConsole` but it is not obvious why we have both and/or that they are not linked...